templates/repo/issue/list.tmpl

```diff
@@ -81,15 +81,15 @@
 				{{end}}
 			{{end}}
 		</div>
-		<div id="issue-filters" class="issue-list-toolbar">
-			<div class="issue-list-toolbar-left">
+		<div id="issue-filters" class="ui stackable grid">
+			<div class="six wide column issue-list-toolbar-left">
```

Comment:
I am not sure whether it needs "stackable grid", it looks like almost every style is defined by our code now, but not by Fomantic's mechanism?

Reply:
`stackable ` Removed.